browser/conftest.py

```diff
@@ -17,7 +20,37 @@
 TEST_DATABASE_URI = 'sqlite:///{}'.format(TESTDB_PATH)
 
 
-@pytest.fixture
+# TODO: Could this become a fixture?
+class DummyLoader(Loader):
+    def __init__(self, raw=None, labels=None, path='test.npz', source='s3'):
+        super().__init__()
+        if raw is None:
+            raw = np.zeros((1, 1, 1, 1))
+        if labels is not None:
+            if raw.shape != labels.shape:
+                raw = np.zeros(labels.shape)
+            self._label_array = labels
+        self._raw_array = raw
+
+        self._path = path
+        self.source = source
+
+        if is_track_file(self._path) and labels is not None:
+            self._label_array = labels[..., [0]]  # .trk files have only one feature
+            self._raw_array = np.zeros(self._label_array.shape)
+
+            self._cell_info = {0:
+                               {label: {'frame_div': None,
+                                        'daughters': [],
+                                        # NOTE: assumes all labels are in all frames
+                                        'frames': list(range(self._label_array.shape[0])),
+                                        'label': label,
+                                        'capped': False,
+                                        'parent': None}
+                                for label in np.unique(self._label_array) if label != 0}}
```

Comment:
Done, I had to expand the functionality of `LabelInfoMaker` to make this happen tho. Previously, we only used `LabelInfoMaker` for ZStack metadata, and this code is trying to spoof tracking metadata. `LabelInfoMaker` now has a `tracking` flag that will make an empty lineage instead when True.

This means we could add support for .trk files with no lineages, or potentially converting zstack projects to tracking projects. Not a change I've made yet, but something to consider

Reply:
Re: converting zstack projects to tracking projects, the desktop version has a `save_as_trk` save option, which adds blank fields for the tracking lineage info and saves in .trk format. In some contexts, this is my preferred method of making tracked files (all seg fixes + assigning correct labels while in .npz format, add lineage info, open as .trk and assign parents where needed). Taking this one step further and actually doing lineage assignments with the .npz, then saving as .trk, is something I've started to consider as well.

I think it would be nice to support "upgrading" the project type partway through working on a file. However, track-editing is currently not as robust as segmentation-editing (not just on the frontend). I'll be looking into making track-editing more robust as I get more into my tracking data sprint, so perhaps we could revisit the idea after that.